content/en/docs/concepts/scheduling-eviction/assign-pod-node.md

```diff
@@ -285,16 +285,42 @@ To use inter-pod affinity, use the `affinity.podAffinity` field in the Pod spec.
 For inter-pod anti-affinity, use the `affinity.podAntiAffinity` field in the Pod
 spec.
 
-#### Scheduling a group of pods with inter-pod affinity to themselves
+#### Existing Pods' Affinity and Anti-Affinity Considerations
+
+When scheduling a new Pod, the Kubernetes scheduler evaluates both the incoming Pod's affinity/anti-affinity rules and those of the existing Pods in the cluster. Here's how these rules are processed:
+
+1. **Hard Constraints (Node Filtering):**
+   - **Incoming Pod's `podAffinity.requiredDuringSchedulingIgnoredDuringExecution`:**
```

Comment:
can you just mention this field and `podAntiAffinity.requiredDuringSchedulingIgnoredDuringExecution` in one sentence, since there's really no special treatment among the two

Reply:
That's a great point! Both podAffinity.requiredDuringSchedulingIgnoredDuringExecution and podAntiAffinity.requiredDuringSchedulingIgnoredDuringExecution are treated similarly as hard constraints during scheduling. I'll combine them into a single sentence to improve clarity:

Incoming Pod's podAffinity or podAntiAffinity with requiredDuringSchedulingIgnoredDuringExecution: Ensures the new Pod is scheduled onto nodes that satisfy these required affinity or anti-affinity rules based on existing Pods.
Thank you for the suggestion—I believe this will make the documentation more concise and easier to understand.

Reply:
It looks like you reverted this change?